model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NamedTests.xtend

```diff
@@ -8,25 +8,387 @@ import org.eclipse.xtext.testing.validation.ValidationTestHelper
+         entity Ent4 {
+         }
+        '''.parse => [
+            m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+        ]
+    }
+
+    @Test
+    @TestCase("TC021")
+    @Requirement(reqs = #[
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-MDL-001",
+            "REQ-MDL-003",
+            "REQ-TYPE-001",
+            "REQ-TYPE-002",
+            "REQ-TYPE-003",
+            "REQ-TYPE-004",
+            "REQ-TYPE-005",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-TYPE-009",
+            "REQ-ENT-001",
+            "REQ-ENT-008"
+    ])
+    def void testNamingOfDerived() {
+        '''
+         model modelTC021;
+
+         import judo::types;
+
+         type string SType(min-size = 0, max-size = 10);
+         type numeric Integer(precision = 9, scale = 0);
+
+         enum Enum {
+              a0 = 0;
+              b1 = 1;
+         }
+
+         entity Ent1 {
+             derived Boolean xBool => true;
+             derived SType xString => "Sting";
+             derived Integer xNumeric => 1;
+             derived Enum xEnum => Enum#a0;
+             derived Date xDate => `2023-11-09`;
+             derived Timestamp xTimestamp => `2020-02-18T10:11:12`;
+             derived Time xTime => `12:23:56.1`;
+         }
+         '''.parse => [
+             m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+         ]
+    }
+
+        @Test
+        @TestCase("TC021")
+        @Requirement(reqs = #[
+                "REQ-SYNT-001",
+                "REQ-SYNT-002",
+                "REQ-SYNT-003",
+                "REQ-SYNT-004",
+                "REQ-MDL-001",
+                "REQ-MDL-003",
+                "REQ-TYPE-001",
+                "REQ-TYPE-002",
+                "REQ-TYPE-004",
+                "REQ-TYPE-005",
+                "REQ-TYPE-006",
+                "REQ-TYPE-007",
+                "REQ-TYPE-008",
+                "REQ-TYPE-009",
+                "REQ-ENT-001",
+                "REQ-ENT-009",
+                "REQ-ENT-011"
+        ])
+        def void testNamingOfQuery() {
+            '''
+             model modelTC021;
+
+             import judo::types;
+
+             type string SType(min-size = 0, max-size = 10);
+             type numeric Integer(precision = 9, scale = 0);
+
+             enum Enum {
+                  a0 = 0;
+                  b1 = 1;
+             }
+
+             entity Ent1 {
+                 query Boolean xBool() => true;
+                 query SType xString() => "Sting";
+                 query Integer xNumeric() => 1;
+                 query Enum xEnum() => Enum#a0;
+                 query Date xDate() => `2023-11-09`;
+                 query Timestamp xTimestamp() => `2020-02-18T10:11:12`;
+                 query Time xTime() => `12:23:56.1`;
```

Comment:
Wher the multiple errors are tested? It only match for one of an error

Reply:
https://github.com/BlackBeltTechnology/judo-meta-jsl/pull/140/commits/c21043a6bddf93d2553d733773942aa934874006